lib/online_migrations/change_column_type_helpers.rb

```diff
@@ -247,22 +248,7 @@ def finalize_columns_type_change(table_name, *column_names)
 
           # At this point we are sure there are no NULLs in this column
           transaction do
-            # For PG < 11 and Primary Key conversions, setting a column as the PK
-            # converts even check constraints to NOT NULL column constraints
-            # and forces an inline re-verification of the whole table.
```

Comment:
The comments above seemed to belong closer to the '< 11' && PK codepath in `initialize_columns_type_change` so moved there.

The rest of the comments below were moved into the new `__set_not_null` private method.